src/Sarif.Multitool.Library/Rules/SARIF2006.UrisShouldBeReachable.cs

```diff
@@ -91,7 +91,9 @@ private void AnalyzeUri(string uriString, string pointer)
             {
                 // Ok, it's a well-formed absolute URI. If it's not reachable, _now_ we can report it.
                 Uri uri = new Uri(uriString, UriKind.Absolute);
-                if (!IsUriReachable(uri.AbsoluteUri))
+                if (uri.AbsoluteUri != "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json" &&
```

Comment:
>uri.AbsoluteUri != "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json" [](http://example.com/codeflow?start=20&length=94)

We need this hard code to prevent many test failures before we upload the file to schemastore.
Changing only here is easier to revert, than changes at all failure places.

Reply:
OK! When you go back to this consider whether this logic should permit a larger set of schema URLs and/or whether we could remove it entirely (by making this URL reachable).